Source/WebKit/WebProcess/WebPage/wpe/WebPageWPE.cpp

```diff
@@ -48,38 +48,34 @@ bool WebPage::platformCanHandleRequest(const ResourceRequest&)
 
 bool WebPage::hoverSupportedByPrimaryPointingDevice() const
 {
-#if ENABLE(TOUCH_EVENTS)
-    return !screenIsTouchPrimaryInputDevice();
-#else
-    return true;
-#endif
+    // For now, there's no concept of primary vs any
+    return WebProcess::singleton().availableInputDevices().contains(AvailableInputDevices::Mouse);
```

Comment:
We can add WebProcess::primaryPointingDevice() and for now just return mouse if present, otherwise touchscreen if available or none